spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/TableWriter.scala

```diff
@@ -49,15 +47,15 @@ class TableWriter[T] private (
     val valueSpec = quotedColumnNames.map(":" + _).mkString(", ")
 
     val ttlSpec = writeConf.ttl match {
-      case x: PerRowWriteOption[Int] => Some(s"TTL :${x.placeholder}")
-      case x: StaticWriteOption[Int] => Some(s"TTL ${x.value}")
-      case TTLOption.auto => None
+      case TTLOption(PerRowWriteOptionValue(placeholder)) => Some(s"TTL :$placeholder")
+      case TTLOption(StaticWriteOptionValue(value)) => Some(s"TTL $value")
+      case _ => None
     }
 
     val timestampSpec = writeConf.timestamp match {
-      case x: PerRowWriteOption[Long] => Some(s"TIMESTAMP :${x.placeholder}")
-      case x: StaticWriteOption[Long] => Some(s"TIMESTAMP ${x.value}")
-      case TimestampOption.auto => None
+      case TimestampOption(PerRowWriteOptionValue(placeholder)) => Some(s"TIMESTAMP :$placeholder")
+      case TimestampOption(StaticWriteOptionValue(value)) => Some(s"TIMESTAMP $value")
+      case _ => None
```

Comment:
feels repetitive with lines 50 and 51. Like we need a simple function to abstract Some(s"xx ...") for both vals